functests/1_performance/performance.go

```diff
@@ -781,6 +782,19 @@ func validatTunedActiveProfile(nodes []corev1.Node) {
 	var err error
 	var out []byte
 	activeProfileName := components.GetComponentName(testutils.PerformanceProfileName, components.ProfileNamePerformance)
+
+	// check if some another Tuned profile overwrites PAO profile
```

Comment:
Why not skip the test in this case?

Reply:
this function _validatTunedActiveProfile()_ is a part of several tests where we are also verifying some system configurations. Most of users should not change these configurations, so I'm not sure skipping the whole test is a good idea.. User will be warned that this test may fail, but in most of scenarios it should be passed.